backend/src/committee_session/status.rs

```diff
@@ -137,10 +137,8 @@ impl CommitteeSessionStatus {
             CommitteeSessionStatus::Created => Ok(self),
             CommitteeSessionStatus::DataEntryNotStarted
             | CommitteeSessionStatus::DataEntryInProgress
-            | CommitteeSessionStatus::DataEntryPaused => Ok(CommitteeSessionStatus::Created),
-            CommitteeSessionStatus::DataEntryFinished => {
-                Err(CommitteeSessionError::InvalidStatusTransition)
-            }
+            | CommitteeSessionStatus::DataEntryPaused
+            | CommitteeSessionStatus::DataEntryFinished => Ok(CommitteeSessionStatus::Created),
         }
     }
```

Comment:
From what I can tell `prepare_data_entry` is only ever called when the existing status is `CommitteeSessionStatus::Created`:

https://github.com/kiesraad/abacus/blob/1771f092b0a7d6e665ac33af7849c7d0feda4e65/backend/src/committee_session/status.rs#L60

So I'm not sure what effect this change has, can you explain the change a bit more?